bin/release

```diff
@@ -36,8 +36,11 @@ else
 fi
 
 echo "Diffing ENV variables between staging and production."
-diff <(heroku config -r staging) <(heroku config -r production) && echo ""
-read -p "Are these config variables the same (ignoring their value) (y/N) " -n 1 -r
+staging_env=$(heroku config -r staging --json | jq . -S | jq 'keys')
+production_env=$(heroku config -r production --json | jq . -S | jq 'keys')
+echo ${staging_env[@]} ${production_env[@]} | tr ' ' '\n' | sort | uniq -u
+echo "Any keys listed above exist on one env but not the other."
```

Comment:
may want to be explicit about exist on staging but not on prod

Reply:
and maybe have a note saying "if this is unexpected, make sure to add the environment variable to production"